docs/tutorial/security.md

```diff
@@ -761,7 +762,47 @@ function validateSender (frame) {
 }
 ```
 
-### 18. Avoid usage of the `file://` protocol and prefer usage of custom protocols
+### 18. Do not expose Electron's APIs to untrusted web content
+
+You should not directly expose Electron's APIs, especially IPC, to untrusted web content in your
+preload scripts.
+
+### Why?
+
+Exposing raw APIs like `ipcRenderer.on` is dangerous because it gives renderer processes direct
+access to the entire IPC event system, allowing them to listen for any IPC events, not just the ones
+intended for them. To avoid that exposure, we also cannot pass callbacks directly through: The first
```

Comment:
Can we break the callback issue into a separate paragraph?

What I dislike about the current docs (pre this PR) is that the callback issue is hidden in a small side note. It's easy to miss. I first did.

I think it should be very prominent here so that people don't miss it. It isn't trivial for someone to figure out on their own if they aren't familiar with Electron's internals. So it's easy for this to slip through unless it's really prominent in the docs.